packages/core/src/Stores/Modules/Cashier/on-ramp-store.js

```diff
@@ -33,9 +33,21 @@ export default class OnRampStore extends BaseStore {
                         this.default_from_currency
                     }&toDefault=${currency}&theme=danger&merchant_id=iiq3jdt2p44yrfbx&payment_id=&v=2" width="100%" height="475px" class="changelly" scrolling="no" onLoad="function at(t){var e=t.target,i=e.parentNode,n=e.contentWindow,r=function(){return n.postMessage({width:i.offsetWidth},it.url)};window.addEventListener('resize',r),r()};at.apply(this, arguments);" style="min-height: 100%; min-width: 100%; overflow-y: visible; border: none">Can't load widget</iframe>`;
                 },
-                icon: 'IcCashierChangelly',
+                icon: isMobile() ? 'IcCashierChangellyMobile' : 'IcCashierChangelly',
                 name: 'Changelly',
-                payment_icons: ['IcCashierVisa', 'IcCashierMastercard'],
+                payment_icons: [
+                    {
+                        icon: 'IcCashierVisa',
+                        dark_mode_fill: '#ffffff',
+                        light_mode_fill: '#1A1F71',
+                        custom_color: true,
```

Comment:
Use `is_`, `has_`, `should_` prefixes for booleans.

```suggestion
                        has_custom_color: true,
```